web/modules/custom/profile_block/src/Controller/controller.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * @file
+ * @author
+ * Contains \Drupal\profile_block\Controller\controller.
+ * Please place this file under your example(module_root_folder)/src/Controller/
+ */ 
+namespace Drupal\profile_block\Controller;
+/**
+ * Provides route responses for the Example module.
+ */
+class controller{
```

Comment:
Remove the empty file

Reply:
removed this file in PR 87: https://github.com/guptahemant/insta-clone/pull/87